common/lib/xmodule/xmodule/modulestore/split_mongo/split_draft.py

```diff
@@ -198,6 +198,8 @@ def delete_item(self, location, user_id, revision=None, skip_auto_publish=False,
                 branches_to_delete = [ModuleStoreEnum.BranchName.published, ModuleStoreEnum.BranchName.draft]
             elif revision is None:
                 branches_to_delete = [ModuleStoreEnum.BranchName.draft]
+                if location.category in DIRECT_ONLY_CATEGORIES:
+                    branches_to_delete.append(ModuleStoreEnum.BranchName.published)
```

Comment:
My understanding of the bug was that the about module wasn't being auto-published when you deleted one of its children:
```
   about
     |
   video
```
So if you deleted `video`, the published `about` wasn't being updated.
In fact, when we fix this bug, we should encounter https://openedx.atlassian.net/browse/PLAT-799.